index.html

```diff
@@ -11,11 +11,11 @@
     <table class="table">
```

Comment:
Hello @tajugithub85 well done for making an attempt. You are close. You need to put the details in the  `tbody` not in the `thead` section of the code. Please make changes and resubmit.